test/ng/directive/inputSpec.js

```diff
@@ -4211,6 +4211,22 @@ describe('input', function() {
       expect(inputElm[0].getAttribute('value')).toBe('something');
     });
 
+    it('should update the dom "value" to "" and attribute to null when the binding is set to undefined', function() {
+      var inputElm = helper.compileInput('<input type="text" ng-value="value">');
+
+      $rootScope.$apply('value = \'something\'');
```

Comment:
Nit: Feel free to use `"` inside string expressions :grin: (E.g.: `'value = "something"'`)